tests/tests/test_github_webhooks.py

```diff
@@ -0,0 +1,605 @@
+        "info:deviceconfig version origin/master is being used in master",
+        "info:deviceconnect version origin/master is being used in master",
+        "info:gui version origin/master is being used in master",
+        "info:inventory version origin/master is being used in master",
+        "info:inventory-enterprise version origin/master is being used in master",
+        "info:mender version origin/master is being used in master",
+        "info:mender-artifact version origin/master is being used in master",
+        "info:mender-cli version origin/master is being used in master",
+        "info:mender-connect version origin/master is being used in master",
+        "info:mtls-ambassador version origin/master is being used in master",
+        "info:tenantadm version origin/master is being used in master",
+        "info:useradm version origin/master is being used in master",
+        "info:useradm-enterprise version origin/master is being used in master",
+        "info:workflows-enterprise version origin/master is being used in master",
+        "info:syncIfOSHasEnterpriseRepo: Merge to (master) in an OS repository detected. Syncing the repositories...",
+        "git.Run: /usr/bin/git init .",
+        "git.Run: /usr/bin/git remote add opensource git@github.com:/mendersoftware/workflows.git",
+        "git.Run: /usr/bin/git remote add enterprise git@github.com:/mendersoftware/workflows-enterprise.git",
+        "git.Run: /usr/bin/git remote add mender-test-bot git@github.com:/mender-test-bot/workflows-enterprise.git",
+        "git.Run: /usr/bin/git config --add user.name mender-test-bot",
+        "git.Run: /usr/bin/git config --add user.email mender@northern.tech",
+        "git.Run: /usr/bin/git fetch opensource master",
+        "git.Run: /usr/bin/git fetch enterprise master:mergeostoent_140",
+        "git.Run: /usr/bin/git checkout mergeostoent_140",
+        "debug:Trying to Merge OS base branch: (master) including PR: (140) into Enterprise: (master)",
+        "git.Run: /usr/bin/git merge -m Merge OS base branch: (master) including PR: (140) into Enterprise: (master) opensource/master",
+        "git.Run: /usr/bin/git push --set-upstream mender-test-bot mergeostoent_140",
+        "info:Merged branch: opensource/workflows/master into enterprise/workflows/master in the Enterprise repo",
+        "github.CreatePullRequest: "
+        'org=mendersoftware,repo=workflows-enterprise,pr={"title":"[Bot] Improve '
+        'logging","head":"mender-test-bot:mergeostoent_140","base":"master","body":"Original '
+        "PR: https://github.com/mendersoftware/workflows/pull/140\\n\\nChangelog: none\\r\\n\\r\\nSigned-off-by: Fabio Tranchitella "
+        '\\u003cfabio.tranchitella@northern.tech\\u003e","maintainer_can_modify":true}',
+        "info:syncIfOSHasEnterpriseRepo: Created PR: 0 on Enterprise/workflows/master",
+        "debug:syncIfOSHasEnterpriseRepo: Created PR: "
```

Comment:
Maybe we could tidy a bit this debug log, I don't think we need the full response. ID, Number and Title should be enough.